shared/src/test/java/io/crate/concurrent/CompletableFuturesTest.java

```diff
@@ -22,23 +22,27 @@
 
 package io.crate.concurrent;
 
-import io.crate.test.integration.CrateUnitTest;
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.TimeUnit;
 
-import static org.hamcrest.Matchers.*;
+public class CompletableFuturesTest extends Assert {
```

Comment:
I did that because CrateUnitTest derived from that as well, but it only contains static methods so I'll remove it.